requirements_all.txt

```diff
@@ -1243,6 +1243,9 @@ simplepush==1.1.4
 # homeassistant.components.alarm_control_panel.simplisafe
 simplisafe-python==1.0.5
 
+# homeassistant.components.sisyphus
+sisyphus-control==2.0.0
```

Comment:
Run `gen_requirements.py` again to update the requirements file. Then the build should pass.